pkg/rekor/verifier.go

```diff
@@ -23,6 +23,8 @@ import (
 	"os"
 	"time"
 
+	ct "github.com/google/certificate-transparency-go"
```

Comment:
Can we create another package, like `pkg/ct`, for handling CT verification, since this isn't for rekor?